bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -629,6 +619,38 @@ for inj_file, tag in zip(inj_files, inj_tags):
                                 tags=[tag])
 
 
+######################## Make combined injection plots ##########################
+# FIXME: Should this be censored_veto?
+if len(files_for_combined_injfind) > 0:
```

Comment:
Well .. should it be censored_veto?  Is the foreground censor file now being created, and if so can it be fed to the injection finding?  (I think/hope the answer is yes.)

Reply:
(Same applies to line 540 above)

Reply:
So I added this because there's some weirdness in the 2-ifo workflow about what jobs get the censored veto, and which do not. I also think that the answer is "yes", but that is not always the case in the old workflow. Perhaps there is a reason or this, but maybe it is a bug or oversight.

Reply:
should I revert this change then?

Reply:
I think this 'fixme' confusion is a relic of when the fg_censor job wasn't being done in the multi-ifo code because it was broken. Now that it isn't (fingers crossed) we *should* be using fg_censor to veto injections so that we don't erroneously label a missed injection right next to a loud full_data event as found.